src/module/token/minting/MintableERC721.sol

```diff
@@ -148,7 +119,7 @@ contract MintableERC721 is Module, EIP712, BeforeMintCallbackERC721, OnTokenURIC
     //////////////////////////////////////////////////////////////*/
 
     bytes32 private constant TYPEHASH_MINTABLE_ERC721 = keccak256(
```

Comment:
Remove

Reply:
addressed in this PR
https://github.com/thirdweb-dev/modular-contracts/pull/145